test/modules/linear_boltzmann_solvers/transport_steady_cbc/hdpe_balance.lua

```diff
@@ -0,0 +1,79 @@
+-- Infinite 1-group problem
+-- Create Mesh
+nodes={}
+N=2
+L=10
+xmin = -L/2
+dx = L/N
+for i=1,(N+1) do
+  k=i-1
+  nodes[i] = xmin + k*dx
+end
+
+meshgen = mesh.OrthogonalMeshGenerator.Create({node_sets = {nodes,nodes,nodes}})
+mesh.MeshGenerator.Execute(meshgen)
+
+-- Set Material IDs
+mesh.SetUniformMaterialID(0)
+
+materials = {}
+materials[1] = mat.AddMaterial('HDPE') 
+
+num_groups=1
+
+-- Add cross sections to materials
+if num_groups == 172 then
+  mat.SetProperty(materials[1], TRANSPORT_XSECTIONS, OPENMC_XSLIB, "../../MGXS_Generation/detector/mgxs_det/mgxs_detector.h5", 294.0, "hdpe")
+else
+  total = {1.}
+  c     = {0.9}
+  mat.SetProperty(materials[1], TRANSPORT_XSECTIONS, SIMPLE_ONE_GROUP, total[1], c[1])
+end
```

Comment:
currently, this input file is only using the 1-g manufactured data. 
- If you prefer that the test be run with the real HDPE XS, let me know. I can email them to you. 
- If you prefer to keep the 1-g test case, then you should delete the unused lines and remove the if statement and also, maybe, change the name of the lua file so that it does not refer to HDPE.

Reply:
Yes, I was going to ask you to send me your HDPE XS file. If you'll email it to me, I'll fix these tests.